content/docs/1.11.0/v2-data-engine/quick-start.md

```diff
@@ -331,6 +331,14 @@ losetup -j blockfile
 
 #### Add disks to `node.longhorn.io`
 
+> **Note:**
+>
+> Starting from version 1.11.0, Longhorn prevents adding block disks that contain an existing filesystem or partition table to avoid unexpected data loss. Ensure the disk is clean before adding it by running:
```

Comment:
```suggestion
> Starting with version 1.11.0, Longhorn prevents adding block disks that contain an existing file system or partition table to avoid unexpected data loss. Ensure the disk is clean before adding it by running:
```